bitcoin/src/amount.rs

```diff
@@ -83,8 +83,28 @@ impl Denomination {
             Denomination::MilliSatoshi => "msat",
         }
     }
+
+    /// The different str forms of denominations that are recognized.
+    fn forms(s: &str) -> Option<Self> {
+        match s {
+            "BTC" | "btc" | "BTCs" | "btcs" => Some(Denomination::Bitcoin),
+            "cBTC" | "cbtc" | "cBTCs" | "cbtcs" => Some(Denomination::CentiBitcoin),
+            "mBTC" | "mbtc" | "mBTCs" | "mbtcs" => Some(Denomination::MilliBitcoin),
+            "uBTC" | "ubtc" | "uBTCs" | "ubtcs" => Some(Denomination::MicroBitcoin),
+            "nBTC" | "nbtc" | "nBTCs" | "nbtcs" => Some(Denomination::NanoBitcoin),
+            "pBTC" | "pbtc" | "pBTCs" | "pbtcs" => Some(Denomination::PicoBitcoin),
+            "bit" | "bits" => Some(Denomination::Bit),
```

Comment:
If we accept "SAT" I think we should accept "BIT"
```suggestion
            "bit" | "bits" | "BIT" | "BITS" => Some(Denomination::Bit),
```